src/shared/Helpers.js

```diff
@@ -15,7 +15,7 @@ export const makeMapId = () => {
 };
 
 export const getNewId = () => {
-  return Math.floor((new Date().getTime() + Math.random()) * 10);
+  return Math.floor((new Date().getTime() + Math.random()) * 1000); // random * 1000 makes difference to the newId
```

Comment:
Javascript integers are only precise up to 15 digits. Multiplying by 1000 gives a 16 digit number. Multiplying by 10 is only a 14 digit number. I don't remember why we did 14 instead of 15. Either way 16 digits is too long so we need to leave it at multiplying by 10 for now. This really should be a string but we haven't changed everything over yet.